.gitignore

```diff
@@ -4,3 +4,4 @@ composer.lock
 .php_cs.cache
 .DS_Store
 Thumbs.db
+.idea/
```

Comment:
Mess? How often do you need to peruse the ignore file? And there aren't that many IDEs... I wouldn't argue that you go through and exhaustively add all IDE ignores; but I mean if a developer comes in using a particular IDE, why not? As long as the ignore spec won't interfere with something meaningful, it will certainly never have any performance or code readability implications.